File: C++/Daily Flash/Week2/Day6_Jan26/code/prog1.cpp

#include <iostream>

int main() {
	int num1,num2;
	std::cout << "Enetr range for printing odd num------------\n";
	std::cout << "Enter num1: ";
	std::cin >> num1;
	
	std::cout << "Enter num2: ";
	std::cin >> num2;

	while(num2< num1) {
		std::cout <<"Num2 should be greater than num1 \nEnter num2 again : ";
		std::cin >> num2;
	}
	for(int i=num1; i<=num2; i++){
		if(i%2!=0)
			std::cout << i << " ";
	}

	std::cout << std::endl;
        return 0;
}

File: JAVA/Daily Flash Codes/Week2/Day6_Jan25/code/prog1.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int num1,num2;
		System.out.println("Enter range to print odd num: ");
		System.out.println("Enter num1: ");
		num1 = 	Integer.parseInt(in.readLine());
		
		System.out.println("Enter num2: ");
		num2 = 	Integer.parseInt(in.readLine());
			
		while(num2 < num1){
			System.out.println("num1 should be greater than num1\nEnter num2: ");
			num2 = 	Integer.parseInt(in.readLine());
		}

		for(int lc=num1; lc<=num2; lc++) {
			if(lc%2!=0)
				System.out.print(lc+ "  ");
		}
		System.out.println();	
	 
	}


}
